helpers/integration.go

```diff
@@ -289,3 +291,43 @@ func HasPipelineRunFinished(object client.Object) bool {
 
 	return false
 }
+
+func IsEnvironmentEphemeral(testEnvironment *applicationapiv1alpha1.Environment) bool {
+	isEphemeral := false
+	for _, tag := range testEnvironment.Spec.Tags {
+		if tag == "ephemeral" {
+			isEphemeral = true
+			break
+		}
+	}
+	return isEphemeral
+}
+
+func CleanUpEphemeralEnvironments(client client.Client, logger *IntegrationLogger, ctx context.Context, env *applicationapiv1alpha1.Environment,
+	dt *applicationapiv1alpha1.DeploymentTarget, dtc *applicationapiv1alpha1.DeploymentTargetClaim) error {
+
+	logger.Info("Deleting deploymentTarget", "deploymentTarget.Name", dt.Name)
+	err := client.Delete(ctx, dt)
```

Comment:
I just realized that we shouldn't need to delete the `deploymentTarget` based on [this document](https://hackmd.io/UNMAvSdxSeezyliDk3bbzg#Scenario-Deprovisioning-DTDTC), @Josh-Everett are you ok with removing it as part of this refactor?

Reply:
So it sounds like a DTC will have reclaimPolicy field in its spec that will determine if the DT is reclaimed or deleted? When the DTC gets a deleted timestamp the DT reconciler will deal with the reclaiming/deletion? Sounds like the DT delete needs to be removed from this code and the DT controller will deal with the rest.

Reply:
Yes, exactly. It's also possible deleting DT could cause unpredictable race conditions, so best to not delete it explicitly.